nest/03.nest.md

```diff
@@ -0,0 +1,89 @@
+# 2 nest
+In this chapter, we will detail how to use the binary `nest`.
+As explained before, this binary is mainly designed for ease of use.
+However, anyone can use it.
+
+## 2.1 nest options
+`nest` has three options available:
+
+* `-h` or `--help`, which prints the help information of `nest`, consisting of a short explanation for each of `nest`'s options and commands.
+* `-v`, which sets the level of verbosity for `nest`. Basically, how much `nest` will explain what it's doing in the terminal.
+* `-V` or `--version`, which prints the version of `nest` you are currently using.
+
+To use them, simply type `nest` followed by any option. For example, `nest -h`.
+
+## 2.2 nest commands
+### 2.2.1 Updating the list of available packages with `nest pull`
+This command synchronizes the cache of local packages with the one on Raven's server.
+In other words, `nest pull` updates the list of available packages for you to install with Nest.
+You have to run this command frequently (at least before installing or upgrading a package) in order to have an up to date cache.
+Otherwise, you might not be able to install new packages, or new versions of packages that were uploaded after the last time you ran `nest pull`.
+
+#### Common issues
+* `error: unable to pull repository stable: no working mirror found`
+
+[//]: # (TODO: add link to the section 3.1.3)
+Basically, this error means that `nest` couldn't pull the repository `stable`.
+The most common reason is that there's an issue with your Internet connection.
+It might also be that all the mirrors in the repositories configuration file are down, so you need to add working mirrors to the list.
+You can have more information about this file and mirrors in [the section 3.1.3]().
+
+### 2.2.2 Installing and uninstalling a package
+#### Installing a package with `nest install`
+[//]: # (TODO: add link to the section 1.4)
+`nest install` is the command to install a package.
+To do so, run `nest install <package>` in a terminal.
+Don't forget to use the syntax explained in [the section 1.4]() for the package name!
```

Comment:
I'm not a huge linguist, but aren't you supposed to say `Use the syntax explained in section 1.4` instead of `in the section 1.4`?

Reply:
I think the first one is correct. I was actually hesitant to propose this because I figured that the sentence doesn't sound to bad, but if you think so too it's probably a sign it should change.